src/tests/resistance/post_handshake/random_bytes.rs

```diff
@@ -0,0 +1,258 @@
+use bytes::{BufMut, BytesMut};
+use tempfile::TempDir;
+use tokio::time::sleep;
+
+use crate::{
+    protocol::codecs::{payload::Payload, tagmsg::Tag},
+    setup::node::Node,
+    tests::resistance::WAIT_FOR_DISCONNECT,
+    tools::{
+        constants::{
+            ERR_NODE_ADDR, ERR_NODE_BUILD, ERR_NODE_CONNECT, ERR_NODE_STOP, ERR_SYNTH_BUILD,
+            ERR_SYNTH_UNICAST, ERR_TEMPDIR_NEW,
+        },
+        synthetic_node::SyntheticNodeBuilder,
+        util::gen_rand_bytes,
+    },
+};
+
+/// Send given bytes directly to the node after the handshake and return the connection status.
```

Comment:
```suggestion
/// Send given bytes directly to the node after the handshake and returns the connection status.
```

Reply:
this was actually fine, unless `Send` is changed to `Sends` too

Reply:
Indeed, missed the first one. The original is actually more consistent with the naming practice repo-wide so ignore my previous suggestion 😄